gems/pending/postgres_ha_admin/database_yml.rb

```diff
@@ -0,0 +1,68 @@
+require 'util/miq-password'
+
+module PostgresHaAdmin
+  class DatabaseYml
+    attr_reader :db_yml_file, :environment
+
+    def initialize(db_yml_file, environment)
+      @db_yml_file = db_yml_file
+      @environment = environment
+    end
+
+    def pg_params_from_database_yml
+      rails_params_to_pg(params_from_database_yml)
+    end
+
+    def update_database_yml(params)
+      db_yml = YAML.load_file(db_yml_file)
```

Comment:
It looks like you're calling `YAML.load_file` twice, once here and once in `params_from_database_yml`

I assume this one should have been removed?